doc/Manuals/ig-cs_x-road_6_central_server_installation_guide.md

```diff
@@ -1,63 +1,66 @@
 # X-Road: Central Server Installation Guide <!-- omit in toc -->
 
-Version: 2.40
+Version: 2.41
```

Comment:
The installation guide doesn't explain what needs to be done to switch the global configuration sign key type from RSA to EC. It should be explained what's the default key type and how it can be changed. Also, the documentation should explain how changing the key type may affect backwards compatibility and interoperability between two X-Road instances in a federated setup. Probably the best approach is to add the documentation to the Central Server user guide and add a link to it in the Central Server installation guide.

Reply:
Updated